### DISCUSSION TITLE
Different issue templates

### DISCUSSION THREAD

**Discussion by Faq** [TrinityCore/TrinityCore#25949] [Suggestions]
Would be nice for TC repo to use more issue template types what offers [github](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository).

Made some simple example **[here](https://github.com/Faq/Truice-FMX/issues/new/choose)**

![image](https://user-images.githubusercontent.com/322016/105735631-98226700-5f3c-11eb-9e4f-4ef24b71dfe6.png)

**Comment by jackpoz** [comment]
@Aokromes summoning you

I personally like to have a short description to guide the users and the image above looks super

**Comment by Faq** [comment]
Updated form https://github.com/TrinityCore/TrinityCore/pull/26639

**Comment by jackpoz** [comment]
We plan on using multiple issue templates, testing the new issue forms and seeing how they work first

**Comment by jackpoz** [comment]
https://github.com/TrinityCore/TrinityCore/pull/26651 speeding up the creating of "SQL Fix" form as it was asked and asked and asked

**Comment by Aokromes** [comment]
feature request and crash report, the 2nd with priority high can be interesthing.
